sys/net/gnrc/link_layer/lwmac/lwmac_internal.c

```diff
@@ -31,35 +31,18 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-uint32_t _phase_to_ticks(uint32_t phase)
-{
-    uint32_t rtt_now = rtt_get_counter();
-    uint32_t phase_now = _ticks_to_phase(rtt_now);
-
-    /* Start of current interval */
-    rtt_now -= phase_now;
-
-    /* Phase only in next interval */
-    if (phase < phase_now) {
-        rtt_now += RTT_US_TO_TICKS(LWMAC_WAKEUP_INTERVAL_US);
-    }
-
-    /* Advance to phase */
-    return (rtt_now + phase);
-}
-
-gnrc_mac_tx_neighbor_t *_next_tx_neighbor(gnrc_netdev_t *gnrc_netdev)
+gnrc_mac_tx_neighbor_t *_gnrc_lwmac_next_tx_neighbor(gnrc_netdev_t *gnrc_netdev)
```

Comment:
Private functions you don't need to prefix (but for debugging it's nice). Also: private functions (those not defined in headers) should be static.